docs/source/release/v5.1.0/muon.rst

```diff
@@ -47,6 +47,7 @@ Improvements
 - Tf asymmetry mode can now be performed on double pulse fits from the muon analysis GUI.
 - Updated :ref:`LoadMuonNexusV2 <algm-LoadMuonNexusV2>` and  :ref:`LoadPSIMuonBin <algm-LoadPSIMuonBin>` to load a list of time zeros into a new property TimZeroList.
 - Updated :ref:`LoadMuonNexusV2 <algm-LoadMuonNexusV2>` and  :ref:`LoadPSIMuonBin <algm-LoadPSIMuonBin>` to add an option to not auto-correct the time by loaded timezero.
+- Added in default group and pair selection when loading grouping files from xml.
```

Comment:
Is this more of a bug fix? I would have thought that on loading a grouping file something should be selected.